calamari_ocr/ocr/evaluator.py

```diff
@@ -207,6 +207,6 @@ def evaluate(_sentinel=None, gt_data=None, pred_data=None, processes=1, progress
 
         # evaluate single lines
         out = parallel_map(Evaluator.evaluate_single_args, [{'gt': gt, 'pred': pred, 'skip_empty_gt': skip_empty_gt} for gt, pred in zip(gt_data, pred_data)],
-                           processes=processes, progress_bar=progress_bar, desc="Evaluation")
+                           processes=1, progress_bar=progress_bar, desc="Evaluation")
```

Comment:
Why did you change the processes to 1? Simply undo?

Reply:
done